### DISCUSSION TITLE
CMake Error at src/gpl/CMakeLists.txt:44 (find_package)

### DISCUSSION THREAD

**Discussion by gkamendje** [The-OpenROAD-Project/OpenROAD#5936] [Q&A]
I was able to run ORFS on `openSUSE Leap 15.5` (WSL2) while modifying the dependency installer script. Trying to apply the same recipe to `openSUSE Leap 15.6` leads to the following error message.
```
By not providing "Findortools.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "ortools", but
  CMake did not find one.

  Could not find a package configuration file provided by "ortools" with any
  of the following names:

    ortoolsConfig.cmake
    ortools-config.cmake

  Add the installation prefix of "ortools" to CMAKE_PREFIX_PATH or set
  "ortools_DIR" to a directory containing one of the above files.  If
  "ortools" provides a separate development package or SDK, be sure it has
  been installed.
```
I have no clue where to start debugging this. I am bulding natively and locally. Here is the build command 
```
[INFO] ./tools/OpenROAD/etc/Build.sh -dir=/home/gkamendje/tmp_compile/OR_myclone/tools/OpenROAD/build -threads=4 -cmake= -D CMAKE_INSTALL_PREFIX=/home/gkamendje/tmp_compile/OR_myclone/tools/install/OpenROAD -D CMAKE_INSTALL_RPATH=/home/gkamendje/tmp_compile/OR_myclone/dependencies/lib:/home/gkamendje/tmp_compile/OR_myclone/dependencies/lib64 -D CMAKE_INSTALL_RPATH_USE_LINK_PATH=TRUE 
```

**Comment by gkamendje** [comment]
This looks like #4137. I will double check and probably close this one if issues are the same.

**Reply by maliberty** [reply]
Note #4137 is not an issue but a discussion and is not progressing because the request to file an issue was no responded to.

**Comment by maliberty** [comment]
or-tools need to be installed.  See https://github.com/The-OpenROAD-Project/OpenROAD/blob/7a67188f991813d986df6f7b180c1c15454292d6/etc/DependencyInstaller.sh#L293 for how it is usually handled.  Note that we are not officially support suse yet so the installer may be incomplete.
